### DISCUSSION TITLE
Fork git repo instead of cloning when import a third-party git repository or using the deploy button.

### DISCUSSION THREAD

**Discussion by craigary** [vercel/vercel#6021] [Ideas]
Hi, I developed a project recently, and it works well with vercel. I would love to see more people using my work, even for newbies who have no idea about git commands, so I wrote a detailed tutorial about how to fork and deploy it on vercel, along with setting up environment variables.

I know the Deploy button should be the best way to one-click deployment, but it only clones the project, not fork. It would be nice if vercel uses "fork" to duplicate. So that they can easily pull and merge my updated code.

Thank you very much, and I would like to see vercel can make some changes to this!

**Comment by mathix420** [comment]
I agree, I have exactly the same use-case with one of my projects.
